doc/release/upcoming_changes/26745.highlight.rst

```diff
@@ -0,0 +1,8 @@
+Interactive examples in the NumPy documentation
+-----------------------------------------------
+
+The NumPy documentation includes a number of examples that
+can now be run interactively using WebAssembly and Pyodide.
```

Comment:
I know this is a stupid remark, but not everyone might know that `WebAssembly and Pyodide` means in your borwser.

```suggestion
can now be run interactively in your browser using WebAssembly and Pyodide.
Comment

```